src/graphqlRequest.test.ts

```diff
@@ -1,6 +1,46 @@
 import { graphqlRequest } from './graphqlRequest'
 
 describe('GraphQLRequest', () => {
+  it('Should request have proper structure', async () => {
```

Comment:
This test is not needed as it's testing how Axios handles its params...

Reply:
It mainly checks if query name is converted into operationName...